src/main/java/com/algorithmia/algorithmHandler/RequestHandler.java

```diff
@@ -0,0 +1,55 @@
+package com.algorithmia.algorithmHandler;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.util.Scanner;
+import org.apache.commons.codec.binary.Base64;
+
+
+final class RequestHandler<INPUT>
+{
+
+    private Scanner input;
+    private JsonParser parser;
+
+    RequestHandler(){
+        this.input = new Scanner(System.in);
+        this.parser = new JsonParser();
+    }
+
+    private Request<INPUT> CreateRequest(String contentType, JsonElement data) throws Exception {
```

Comment:
You can remove all of the `throws Exception` from your methods. In modern Java development, checked exceptions are generally frowned upon. You should be using unchecked exceptions (which don't require specification in your signatures) and wrapping any caught checked exceptions to unchecked exceptions in your code.